packages/grafana-ui/src/components/Tabs/TabsBar.tsx

```diff
@@ -28,6 +28,7 @@ export const TabsBar = forwardRef<HTMLDivElement, Props>(({ children, className,
 const getStyles = (theme: GrafanaTheme2) => ({
   tabsWrapper: css({
     borderBottom: `1px solid ${theme.colors.border.weak}`,
+    overflowY: 'visible',
     overflowX: 'auto',
```

Comment:
This is causing the focus styles to be semi-hidden, I would appreciate some input for this.

Reply:
Can we just remove this overflow rule completely? Don't see why it's necessary.

Reply:
We want to be able to scroll when the tabs do fit the screen.